perllib/FixMyStreet/Map/Buckinghamshire.pm

```diff
@@ -15,11 +15,6 @@ sub layer_names{ [ 'Basemapping2018' ] }
 sub scales {
     my $self = shift;
     my @scales = (
-        '1000000',
-        '500000',
```

Comment:
You need all these (bar maybe 1M) for https://fixmystreet.buckscc.gov.uk/reports/Buckinghamshire I think?